SS_global.tpl

```diff
@@ -1062,21 +1049,28 @@ FINAL_SECTION
       {
         Get_Benchmarks(show_MSY);
         if (mceval_phase() == 0)
-          cout << " finished benchmark" << endl;
+        {
+          cout << "Finished calculating benchmarks" << endl;
+          echoinput << "Finished calculating benchmarks" << endl;
+        }
       }
     }
     if (Do_Forecast >= 0)
     {
-      report5 << "THIS FORECAST FOR PURPOSES OF GETTING DISPLAY QUANTITIES" << endl;
+      report5 << "THIS FORECAST IS FOR PURPOSES OF GETTING DISPLAY QUANTITIES" << endl;
       if (did_MSY > 0)
         show_MSY = 0; //  so to not repeat forecast_report.sso
       Get_Forecast();
-      if (mceval_phase() == 0)
-        cout << " finished forecast" << endl;
+      if (mceval_phase() == 0) {
```

Comment:
Curly braces should be on a new line; but, I am not sure if the automatic formatting does this?